integrations/jenkins/documentation.yaml

```diff
@@ -0,0 +1,77 @@
+exporter_type: included
+app_name_short: Jenkins
+app_name: {{app_name_short}}
+app_site_name: Jenkins
+app_site_url: https://www.jenkins.io/
+exporter_name: the Jenkins exporter
+exporter_pkg_name: jenkins
+exporter_repo_url: https://github.com/jenkinsci/prometheus-plugin
+additional_prereq_info: |
+  {{exporter_name}} exposes Prometheus-format metrics when the [metrics](https://plugins.jenkins.io/metrics/)
+  and [prometheus](https://plugins.jenkins.io/prometheus/) plugins are installed.
+  You can run the following checks to verify that {{app_name_short}} is exposing Prometheus-format metrics.
+
+   *  To determine if {{exporter_name}} is exposing metrics, run the following command:
+
+      <pre class="devsite-click-to-copy">
+      kubectl -n <namespace_name> port-forward <jenkins_pod_name> <jenkins_port>
+      curl localhost:<jenkins_port>/prometheus
+      </pre>
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: jenkins
+    labels:
+      app.kubernetes.io/name: jenkins
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+  + - port: 8080
+      scheme: http
+      interval: 30s
+      path: /prometheus
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: jenkins
```

Comment:
Same comment as in https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/422#discussion_r1062917614 -- No need for the + here and on line 33

And we should say something like [this](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/integrations/airflow/documentation.yaml#L42) to remind users to check this against the labels of their jenkins workload, which can vary depending on how they set it up